crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -546,6 +546,75 @@ impl Output for CliOutput {
         self.print_horizontal_table(rows, header);
     }
 
+    fn print_jobs_tasks(&self, mut jobs: Vec<(JobId, JobDetail)>, worker_map: WorkerMap) {
```

Comment:
This function is basically the same as `print_job_tasks`. Let's unify these two functions and only print the job ID column if there is more than one job.